eval.go

```diff
@@ -2076,6 +2076,11 @@ func (e *callExpr) eval(app *app, args []string) {
 
 		app.ui.cmdAccLeft = acc
 		update(app)
+	case "maps":
+		cleanUp := app.runShell("$PAGER", nil, "$|")
```

Comment:
You should try doing something like what I did in https://github.com/gokcehan/lf/pull/918 to try to support Windows.

It'd be even better if somebody could test it on Windows.

Reply:
You're right! I'm on Mac. Didn't really test on Windows.

Although did a quick search seems pipe should work for windows. We're only invoking pager...